.github/workflows/ci.yml

```diff
@@ -419,14 +414,14 @@ jobs:
         rm -rf .git
     - name: Create archive
       run: |
-        cd ..
-        cp -r rizin rizin-${{ steps.extract_version.outputs.branch}}
```

Comment:
does it create the inner directory with the right version in it (e.g. `rizin-0.1.0`)?

Reply:
For some reason it sets `undefined` as a version:
```
meson-dist/rizin-undefined.tar.xz b7d2c38b9a1b859fe943403770313f548d4163728206aafb386a7b344d50794b
total 5368
-rw-r--r-- 1 runner docker 5490232 Jan 13 11:46 rizin-undefined.tar.xz
-rw-r--r-- 1 runner docker      88 Jan 13 11:46 rizin-undefined.tar.xz.sha256sum
```

Reply:
I guess you should define project version inside main `meson.build` file.

Reply:
yes, right @pelijah . Btw, once we remove acr/make we can just use the version in the meson.build project.